src/js/components/Button/doc.js

```diff
@@ -152,4 +157,6 @@ export const themeDoc = {
     description: 'Any additional style for the Button.',
     type: 'string | (props) => {}',
   },
+  ...themeDocUtils.focusStyle,
+  ...themeDocUtils.normalizeColor('The text color of the label.'),
```

Comment:
`normalizeColor` is being used for border, text color, background, and hover https://github.com/grommet/grommet/blob/master/src/js/components/Button/StyledButton.js

and I don't think it uses 'global.colors.text'.